src/views/AddItem.jsx

```diff
@@ -16,14 +16,57 @@ export function AddItem({ listToken }) {
 	const handleSubmit = async (e) => {
 		e.preventDefault();
 
+		// const isValid = (itemName) => {
+		// 	checkEmpty(itemName);
+		// 	checkDuplicates(itemName);
+		// }
+
+		// const checkEmpty = (itemName) => {
+		// 	itemName.trim().length === 0;
+		// 	setMessage("Error: Please enter your item's name.");
+
+		// }
```

Comment:
When you see a function named `checkEmpty`, you expect it to return a boolean. It's like you're asking a question, "Is this itemName empty or not?". So your function should return a yes or no (true/false) and then set the error message.

Reply:
Or if you decide to set the error message inside you functions, you should also add a conditional check (if statement). 
So you should write 
```
if (itemName.trim().length === 0) {
setMessage("Error: Please enter your item's name.");
}
```

Reply:
Done! Thanks for the suggestions :)